src/main/drivers/adc.c

```diff
@@ -33,6 +33,14 @@
 adc_config_t adcConfig[ADC_CHANNEL_COUNT];
 volatile uint16_t adcValues[ADC_CHANNEL_COUNT];
 
+uint8_t adcChannelByPin(ioTag_t pin)
```

Comment:
The function is called `adcChannelByPin` yet what is passed in is a tag. I suggest:

`uint8_t adcChannelByTag(ioTag_t ioTag)`

One of the advantages of the `_t` naming convention for types is that the declared variable can just take the name of the type, minus the `_t` suffix. Especially useful for function parameters.